mitodl/ocw-studio
# Fix ckeditor console error

#### Pre-Flight checklist


#### What are the relevant tickets?
#1632 

#### What's this PR do?
- Prevents SiteContentForm from loading until its data has been loaded.
- Removes redux-logger, because it is annoying and you can get the same info from redux dev tools

#### How should this be manually tested?
1. Navigate to the page listing
2. Open any page for editing
3. Check that no error is logged in console
4. Check that editing works as expected.
